cli/src/utils.js

```diff
@@ -93,8 +94,61 @@ async function runCommand(cmd, parameters = [], options = {}) {
   });
 }
 
+async function findCRI() {
+  // Resolve the CRI to be used
+  if (await which('docker-compose', { nothrow: true }))
+    return ['docker-compose']
+
+  if (await which('docker', { nothrow: true }))
+    return ['docker', 'compose']
+
+  if (await which('podman-compose', { nothrow: true }))
+    return ['podman-compose']
+
+  if (await which('podman', { nothrow: true }))
+    return ['podman', 'compose']
+
+  throw new Error('Cannot find a valid runtime to run containers');
+}
+
+const runComposeActions = {
+  'docker': {
+    start: ['up', '--build', '--force-recreate', '--remove-orphans', '--no-color'],
+    build: ['build', '--no-cache', '--force-rm'],
```

Comment:
'dev' action is missing here as well
'start' should not have `--build` option only 'dev' have it